documentation/flreconstruct/FLReconstruct.md

```diff
@@ -652,8 +656,8 @@ The  FLReconstruct  script  contains  up   to  five  sections  of  type
 	  from the list of modules  defined in the main configuration file
 	  (default: `"pipeline"`).
 
-A sample configuration script  (commented) is provided in this
-[document](FLReconstruct-3.0.0.conf) (for Falaise 3.0.0).
+A sample configuration script  (commented) is provided in
+[this document](FLReconstruct-3.0.0.conf).
```

Comment:
Is this link going to work? It doesn't work for me in the old version. It's not very clear to me what you would set the services parameters to but maybe this document would make it apparent.